docs/getting_started_jax.md

```diff
@@ -0,0 +1,252 @@
+
+A lot easier to read!
+
+Note that `manual_axes` is always equal to the axes in the `mesh` but not in the `auto = frozenset([...])`.
+
+Also, this will be relaxed in the JAX side soon, but you will be able to use `UNSPECIFIED` on in/out specs and have `auto` axes propagate from inside to outside the shard_map body (and vice-versa).
+
+### Auto partitioners
+
+In progress.
+
+### XLA_DUMP_TO
+
+When specifying the `XLA_DUMP_TO`, you will see an additional `shardy/` directory containing various dumps of the StableHLO program. A lot of them are currently only relevant to the Shardy team to debug issues. The one you should focus on when debugging is `sdy_module_after_sdy_export.mlir` which is the module after propagation finishes and the StableHLO program.
+
+## 2. Why Shardy?
+
+### Readability
+
+As seen above, it's much easier to read the shardings and `shard_maps` and understand how they match what is happening in the JAX code. Similarly GSPMD propagation will give back HLO code - not MLIR which both Shardy and `jax.jit.lower` return.
+
+### Interpretability
+
+We are planning on exposing a feature we call "user priorities" (not in JAX yet!). It allows you to attach a value telling Shardy how important a tensor's dimension sharding is over other constraints in the program.
+
+Higher priorities are defines as lower values (lowest being 0).
+
+```python
+PartitionSpec(None, 'x', 'y', priorities=(None, 0, 1))
+```
+
+Here the sharding of dim 1 on `x` has a higher priority than dim 2 on `y`, meaning dim 1 will be propagated through the program first and then dim 2, meaning any potential sharding conflicts will be explicitly resolved by having `x` propagated first.
+
+This can be helpful for debugging models as well by having you break down your sharding strategies to separate rounds of propagation in Shardy. For example:
+
+* Priority 0: data parallelism
+* Priority 1: megatron
+* Priority 2: ZeRO sharding
+
+## FAQS
+
+Below is a list of questions you may have on various JAX features and capabilities.
+
+### JAX Sharding types
+
+#### What about GSPMDSharding?
+
+`GSPMDSharding` is closely tied to `HloShardingV1`/`HloShardingV2`. As such the type itself won't be supported.
+
+#### How about different logical device ID orderings?
+
+`GSPMDSharding`s were the answer for this, but instead (TBD) JAX will add a `device_ids` field to `NamedSharding`. And then a different `@mesh` value will be added to the `StableHLO` which saves the different device orderings. You'd now just need to give names to this type of sharding while you didn't before.
+
+TODO add example
+
+#### What about PositionalSharding?
+
+This won't be supported. Instead use a `NamedSharding` with `device_ids`.
+
+#### PmapSharding
+
+This won't be supported. Shardy is meant for `jax.jit`, not `jax.pmap`.
+
+### Propagation Questions
+
+Section for questions about what you may see during propagation. It's probably best to refer to our existing content under go/shardy.
+
+TODO
```

Comment:
To be developed.

Reply:
One strategy to potentially unblock submitting partial documentation could be to create GH issues tickets for the creating of certain doc sections and link to it from here, may be useful for some more complicated docs. Something like "This documentation is currently a work-in-progress, the overall status is tracked as #...."